docs/src/app/(public)/(content)/react/components/checkbox-group/demos/form/css-modules/index.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { Form } from '@base-ui-components/react/form';
+import { Fieldset } from '@base-ui-components/react/fieldset';
+import { Field } from '@base-ui-components/react/field';
+import { Checkbox } from '@base-ui-components/react/checkbox';
+import { CheckboxGroup } from '@base-ui-components/react/checkbox-group';
+import styles from './index.module.css';
+
+export default function ExampleCheckboxGroup() {
+  const [loading, setLoading] = React.useState(false);
+  return (
+    <Form
+      className={styles.Form}
+      onSubmit={async (event) => {
+        event.preventDefault();
+        const formData = new FormData(event.currentTarget);
+        setLoading(true);
+        await submitForm(formData);
+        setLoading(false);
+      }}
+    >
+      <Field.Root
+        name="toppings"
+        render={<Fieldset.Root />}
+        className={styles.Fieldset}
+      >
+        <Fieldset.Legend className={styles.Legend}>Extra toppings</Fieldset.Legend>
+        <CheckboxGroup
+          defaultValue={[]}
+          disabled={loading}
+          className={styles.CheckboxGroup}
+        >
+          <label className={styles.Item}>
+            <Checkbox.Root value="anchovies" className={styles.Checkbox}>
+              <Checkbox.Indicator className={styles.Indicator}>
+                <CheckIcon className={styles.Icon} />
+              </Checkbox.Indicator>
+            </Checkbox.Root>
+            Anchovies
+          </label>
```

Comment:
Can each checkbox have its own `Field`? That means they can use `Field.Label`, which prevents selection on double click